.circleci/continue_config.yml

```diff
@@ -139,7 +139,7 @@ jobs:
           name: 'Maven Install'
           command: |
             export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-            ./mvnw clean install ${MAVEN_FAST_INSTALL} -pl 'presto-native-execution' -am
+            for i in $(seq 1 3); do ./mvnw clean install ${MAVEN_FAST_INSTALL} -pl 'presto-native-execution' -am && s=0 && break || s=$? && sleep 10; done; (exit $s)
```

Comment:
Is this a copy-paste from somewhere else? I remember seeing similar logic earlier. Would it be possible to refactor to avoid code duplication?

CC: @majetideepak @kgpai

Reply:
Refactored. Thank you.